lighthouse-core/gather/connections/connection.js

```diff
@@ -96,8 +96,8 @@ class Connection {
       this._callbacks.delete(object.id);
       if (object.error) {
         log.formatProtocol('method <= browser ERR',
-            {method: callback.method, params: object.result}, 'error');
-        callback.reject(object.result);
+            {method: callback.method}, 'error');
+        callback.reject(object.error);
```

Comment:
Also by creating a new Error inline there, anyone can print the stack and at the *very least* get back to this protocol handling bit